README.md

```diff
@@ -228,7 +228,10 @@ sequence to have:
 and re-running the formatter, will produce:
 
 ```erl formatted foobar1
-[Foo, Bar]
+[
+    Foo,
+    Bar
+]
```

Comment:
Hmm.... We should probably re-write the guide because it no longer works as advertised. It's not enough to just remove the newline between `[` and first element.

This is so much complexity in here 😢

Reply:
Good catch.
I fixed the guide.

I think it makes sense to not only take a hint from the first newline, but rather from any newline in the container.